.github/workflows/ci.yml

```diff
@@ -253,6 +262,63 @@ jobs:
           fi
           cmake --install ./build --prefix "$llvm_project_root"/install
 
+      - name: LibC++ Binaries
+        id: libcxx-cache
+        uses: actions/cache@v4
+        with:
+          path: ${{ steps.llvm-parameters.outputs.libcxx-root }}
+          key: libcxx-${{ runner.os }}-${{ matrix.compiler }}-${{ matrix.version }}-${{ steps.llvm-parameters.outputs.llvm-hash }}
+
+      - name: Install LibC++
+        id: libcxx-install
+        if: ${{ steps.libcxx-cache.outputs.cache-hit != 'true'}}
+        shell: bash
+        run: |
+          set -x
+
+          cd ..
+          llvm_project_root="$(pwd)/third-party/llvm-project"
+          if [ ! -d "$llvm_project_root/runtimes" ]; then
+              mkdir -p $llvm_project_root
+              cd $llvm_project_root
+              git config --global init.defaultBranch master
+              git config --global advice.detachedHead false
+              git init
+              git remote add origin https://github.com/llvm/llvm-project.git
+              git fetch --depth 1 origin ${{ steps.llvm-parameters.outputs.llvm-hash }}
```

Comment:
Why are we cloning the same repo again in a subdirectory?

Reply:
It is cloning to the same directory: `third-party/llvm-project`

Reply:
Yes. But again?